src/plugins/intel_cpu/src/graph.h

```diff
@@ -65,9 +66,23 @@ class Graph {
         return m_context->getConfig();
     }
 
+    /**
+     * Obsolete way of creating graph
+     * To enable layout propagation and global memory reuse
+     * two-stage creation should be used instead:
+     * - Init()
+     * - Allocate()
+     */
     template<typename NET>
     void CreateGraph(NET &model, const GraphContext::CPtr context);
```

Comment:
I am just not 100% sure that we do not need some wrapper method for Init() + Activate() calls, because, again, I am not sure whether we will be able to utilize this two-step initialization in scope of CompiledModel / InferRequest, before we actually try to do so.
What do you think?

Reply:
As I can see, after the changes made in this PR, there remain only two places where `CreateGraph` is called: `executor_factory.hpp` and `graph_emitter.hpp`, both of which are just place holders for future functionality, so may be easily changed. Therefore, it looks like `CreateGraph(NET &model, const GraphContext::CPtr context)` can be just removed. However it may require a few changes to the unit tests. Actually may be done in a separate PR, I'm just afraid we simply forget about it.

Reply:
Let me rephrase it. I am not sure we don't want to keep a simplified version of the graph creation. When one (i.e. unit tests) does not want to call two functions explicitly, because one does not care about layout propagation and memory reuse.

Reply:
If we are not sure, then just leave this method as it is. If we are sure that this is absolute, then remove it. As simple as that.